packages/appstore/src/components/wallets-consent-popup/wallets-consent-popup.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {
+    Button,
+    Checkbox,
+    Modal,
+    DesktopWrapper,
+    MobileWrapper,
+    MobileDialog,
+    Div100vhContainer,
+} from '@deriv/components';
+import { localize } from '@deriv/translations';
+import WalletsConsentForm from './wallets-consent-form';
+
+type TWalletsConsentPopupProps = {
+    show_wallet_consent_popup: boolean;
+    setShouldShowWalletConsentPopup: (show_wallet_consent_popup: boolean) => void;
+    is_eu: boolean;
```

Comment:
you can observe this component to get this directly from store instead of getting as props

Reply:
Done 😃 👍🏼